plugin-api/src/main/java/org/hyperledger/besu/plugin/services/tracer/ExtendedOperationTracer.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.plugin.services.tracer;
+
+import org.hyperledger.besu.evm.tracing.OperationTracer;
+import org.hyperledger.besu.plugin.data.BlockBody;
+import org.hyperledger.besu.plugin.data.BlockHeader;
+
+public interface ExtendedOperationTracer extends OperationTracer {
```

Comment:
Since all the interfaces are block related I would recommend getting block in the name.  Perhaps `BlockAwareOperationTracer.`

Reply:
Yes, that is a good name. I've changed it.